style.css

```diff
@@ -1,114 +1,215 @@
+        text-align: center;
+        width: 100%;
+    }
 }
 
 .left{
     background-color: blueviolet;
-    
 }
 
 .right{
     background-color: white;
     flex-direction: column;
+    
 }
-.left h2{
+
+.dls{
+    color: azure;
     position: absolute;
     top:60px;
-    left:60px;
-    color: azure;
 }
-.left h1{
+
+.on{
+    /* border-bottom: 1px solid white; */
     color: azure;
     position: absolute;
     top:100px;
-    left:60px;
-    border-bottom: 1px solid white;
 }
-.left h3{
+
+.app{
     color: azure;
+    padding-top: 10px;
     position: absolute;
     top:150px;
-    left:60px;
-}
-.left p{
-    position: absolute;
-    left:70px;
-    top :220px;
-    padding: 10px;
 }
-#second{
-    padding: 20px;
+
+.para{
+    color: azure;
+    font-family: 'Roboto', sans-serif;
+    left: 50px;
     position: absolute;
-    left: 60px;
-    top:320px;
+    top: 240px;
 }
 
-.right h1{
-    text-align: center;
-    padding: 0 0 20px 0;
-    
+.mail-1,.mail-2,.pass-1,.pass-2{
+    background-color:lightgrey;
+    font-size: 20px;
+    color: grey;
+    height: 40px;
+    padding: 0 5px 5px;
+    width: 100%;
 }
 
-.right form{
-    
-    box-sizing: border-box;
-    width: 50%;
-    padding: 10px 10px;
-    margin: 8px ;
-    
-    
+.img{
+    display: inline;
+    height: 3%;
+    margin-left: -5%; 
+    position: absolute;
+    vertical-align: middle; 
+    width: 2.5%;
 }
 
-form .txt_field{
-    position: relative;
-    background-origin: 2px solid;
-    margin: 30px 0;
+.pass{
+    color: blueviolet;
+    display: flex;
+    font-size: 16px;
+    font-weight: bold;
+    Justify-content: flex-end;
+    width: 100%;
 }
 
-.txt_field input{
-    width: 100%;
-    padding: 0 5px;
-    height: 40px;
-    font-size: 16px;
+.c-submit{
+    align-self: center;
+    color:grey;
+    display: flex;
+    font-size: 20px;
+    justify-content: center; 
+    width: 50%; 
+}
 
+.l-foot{
+    display: flow-root;
+    font-size:17px;
+    justify-content: center;
 }
 
-.txt_field label{
-    position: absolute;
-    top: 50%;
-    left: 5px;
-    font-size: 16px;
+.req{
+    color: blueviolet;
+}
 
+.r-head{
+    display: flex;
+    align-items: center;
+    justify-content: center;
 }
-form .pass{
-    
-    float: right;
-    color: blueviolet;
+
+.right form{
+    align-items: center;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 8px;
+    padding: 10px 10px;
+    width: 50%;
     
 }
 
-input[type = "submit"]{
-    width: 40%;
-    float: right;
-    padding: 5px;
+.txt_field{
+    margin: 10px 0;
+    width: 100%;
 }
 
 .footer{
-    float: right;
-    bottom : 10px;
+    bottom: 0px; 
+    color: grey;
+    display: flex;
+    font-size: 10px;
+    justify-content: center;
     position: absolute;
-    left: 150px;
-    
+    text-align: center;
+    width: 100%;
 }
 
 .header{
+    display: flex;
     float:right;
+    padding-right: 10px;
+    padding-top: 10px;
```

Comment:
Read the difference between px and rem. In the majority of our application, we are using rem instead of px. 
Please understand the difference and use rem instead of px.